### DISCUSSION TITLE
Caroules如何实现定时自动轮播图片功能

### DISCUSSION THREAD

**Discussion by 1184898069** [irihitech/Semi.Avalonia#466] [Q&A] [answered]
轮播图如何实现定时自动轮播图片功能，目前使用DispatcherTimer定时器，请问有原生的自动轮播功能吗，如果没有我该如何获取两个按钮的点击事件
![image](https://github.com/user-attachments/assets/b5c325da-528b-4cf0-aaad-10d1c7ee00ff)

**Comment by rabbitism** [comment, marked as answer]
不需要执行点击，直接调用轮播图的next方法就可以了

**Comment by 1184898069** [comment]
好的，谢谢您的解答












张雯轩
***@***.***






---原始邮件---
发件人: "Dong ***@***.***&gt;
发送时间: 2024年11月15日(周五) 晚上8:07
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [irihitech/Semi.Avalonia] Caroules如何实现定时自动轮播图片功能  (Issue #465)




 
不需要执行点击，直接调用轮播图的next方法就可以了
